--- universalTouchGamepad.js	2026-04-02 14:59:23.493710384 +0000
+++ universalTouchGamepad.js	2026-04-02 14:59:31.471710785 +0000
@@ -441,7 +441,7 @@
                         else if (maxDistance <= 0) { x = 0; y = 0;}
                         handleElement.style.left = `${x + (currentBaseSize - currentHandleSize) / 2}px`;
                         handleElement.style.top = `${y + (currentBaseSize - currentHandleSize) / 2}px`;
-                        if (maxDistance > 0) { updateGamepadAxis(joyConfig.axes[0], x / maxDistance); updateGamepadAxis(joyConfig.axes[1], y / maxDistance * -1); }
+                        if (maxDistance > 0) { updateGamepadAxis(joyConfig.axes[0], x / maxDistance); updateGamepadAxis(joyConfig.axes[1], y / maxDistance); }
                         else { updateGamepadAxis(joyConfig.axes[0], 0); updateGamepadAxis(joyConfig.axes[1], 0); }
                     }
                     base.addEventListener('touchstart', onJoystickTouchStart, { passive: false });
